mockfaster/src/main/java/com/decathlon/tzatziki/utils/MockFaster.java

```diff
@@ -216,31 +216,23 @@ public static void verify(HttpRequest httpRequest, VerificationTimes times) {
         }
     }
 
-    public static void assertHasHeaders(HttpRequest request, Map<String, String> headers) {
-        compareHeaders(Comparison.CONTAINS, request, headers);
+    public static Map<String, String> asMap(List<Header> headers) {
+        return headers.stream().collect(toMap(
+                header -> header.getName().getValue(),
+                header -> header.getValues().get(0).toString()));
```

Comment:
Why limit the values to the first ? If we use comparison with jackson underneath, could we equate a singular value as a list of one value ?

Reply:
#137